### PR TITLE
add Webhelpers

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/webhelpers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ea86f284e929366b77424ba9a89341f43ae8dee3cbeb8702f73bcf86058aa583
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py3k]
```
Comment (scopatz) [inline recipes/webhelpers/meta.yaml]: Can this be `noarch: python` and the `- python <=2.7` below?

Comment (rluria14) [reply recipes/webhelpers/meta.yaml]: Couldn't get it to build with `noarch: python and python <=2.7`. I got python package conflicts.

Comment (scopatz) [reply recipes/webhelpers/meta.yaml]: It looks like markupsafe isn't on Python 2, so did you mean to skip Python 2 here, rather than skipping Python 3?

Comment (scopatz) [reply recipes/webhelpers/meta.yaml]: Yeah, this seems like it should be noarch with python >=3.5

Comment (rluria14) [reply recipes/webhelpers/meta.yaml]: Added noarch and a patch to resolve a python3.x/2.x issue with webhelpers.html involving an import of `urllib`

**Comment by rluria14** [comment]
@conda-forge/staged-recipes

File: recipes/webhelpers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  license_file: "LICENSE"
+  summary: "Web Helpers"
+  doc_url: "https://webhelpers.readthedocs.io/en/latest/"
+  dev_url: "https://bitbucket.org/bbangert/webhelpers/src/trunk/"
+
+extra:
+  recipe-maintainers:
+    - mwilson8
+    - rluria14
+    - oblute
```
Comment (scopatz) [inline recipes/webhelpers/meta.yaml]: @mwilson8 @oblute :wink:

Comment (mwilson8) [reply recipes/webhelpers/meta.yaml]: I agree to be a maintainer

**Pull request merged**